From e801534c5eb8cb049dc480fea48fddddd8415d05 Mon Sep 17 00:00:00 2001
From: Tim S <stahta01@users.sourceforge.net>
Date: Wed, 16 Aug 2017 20:35:10 -0400
Subject: [PATCH] Added build flags to fix build errors.

Added "-Wno-deprecated", "-fcheck-new", and "-std=gnu++11" options to fix compiler error/warnings.

---
 builds/posix/prefix.mingw | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/builds/posix/prefix.mingw b/builds/posix/prefix.mingw
index 8e514d8c33..e4f38a1e6e 100644
--- a/builds/posix/prefix.mingw
+++ b/builds/posix/prefix.mingw
@@ -23,8 +23,8 @@
 endif
 
 # -Wno-unused-variable is used due to unused gpre generated variables
-PROD_FLAGS=-O2  -DMINGW -Wall -Wshadow -Wundef -Wno-long-long -Wno-unused-variable -Wno-sign-compare -Wno-parentheses -Wno-switch -fmessage-length=0 -Dlint -DWIN32_LEAN_AND_MEAN -MMD -mthreads
-DEV_FLAGS=-ggdb -DMINGW -Wall -Wshadow -Wundef -Wno-long-long -Wno-unused-variable -Wno-sign-compare -Wno-parentheses -Wno-switch -fmessage-length=0 -Dlint -DWIN32_LEAN_AND_MEAN -MMD -mthreads
+PROD_FLAGS=-O2  -DMINGW -Wall -Wshadow -Wundef -Wno-long-long -Wno-unused-variable -Wno-sign-compare -Wno-parentheses -Wno-switch -Wno-deprecated -fmessage-length=0 -Dlint -DWIN32_LEAN_AND_MEAN -MMD -mthreads -fcheck-new -std=gnu++11
+DEV_FLAGS=-ggdb -DMINGW -Wall -Wshadow -Wundef -Wno-long-long -Wno-unused-variable -Wno-sign-compare -Wno-parentheses -Wno-switch -Wno-deprecated -fmessage-length=0 -Dlint -DWIN32_LEAN_AND_MEAN -MMD -mthreads -fcheck-new -std=gnu++11
 
 PLATFORM_PATH=os/win32
 
